codegen/src/main/ops/org/nd4j/codegen/ops/Linalg.kt

```diff
@@ -206,6 +208,34 @@ fun Linalg() =  Namespace("Linalg") {
             """.trimIndent()
         }
     }
+
+    Op("tri") {
+        javaPackage = "org.nd4j.linalg.api.ops.custom"
+        javaOpClass = "Tri"
+
+        Arg(DataType.INT, "params") {count = AtLeast(3);"shapes of tensor and sub-diagonal number at and below which the array is filled"; }
+
+        Doc(Language.ANY, DocScope.ALL){
```

Comment:
Missed in the review: add an Output(...) here too so generated signatures are `INDArray`/`SDVariable` not `INDArray[]`/`SDVariable[]` as they are now